src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -333,8 +276,19 @@ public boolean isFinal(Element e) {
         return e.getModifiers().contains(Modifier.FINAL);
     }
 
+    /*
+     * A contemporary JLS term for "package private" or "default access" is
+     * "package access". For example: "a member is declared with package
+     * access" or "a member has package access".
+     *
+     * This is to avoid confusion with unrelated _default_ methods which
+     * appeared in JDK 8.
+     */
     public boolean isPackagePrivate(Element e) {
```

Comment:
Maybe change the method name, in line with the comment?

Reply:
I would refrain from doing it in the context of this PR.